curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -253,23 +253,71 @@ DOM inspector
 
 #### --text--
 
-Which of the following attributes is used to specify the maximum number of characters allowed in an input field?
+Which of the following is the correct way to group related input fields?
 
 #### --distractors--
 
-`getMaxLength`
+```html
+<form>
+  <fieldset>
+    <caption>User information</caption>
+
+    <label for="name">Name: </label>
+    <input type="text" id="name" />
+
+    <label for="email">Email: </label>
+    <input type="email" id="email" />
+  </fieldset>
+</form>
+```
 
 ---
 
-`max`
+```html
+<form>
+  <div>
```

Comment:
I vote we change this something like `span` another wrong answer. Because `div` could be used and is correct but `fieldset` is the better option.